README.md

```diff
@@ -1,13 +1,20 @@
-# jkennedy-research-and-reflection
+# Jacob Kennedy - Research & Reflection
 
+## Course goals
+
+<!-- Context 1 -->
```

Comment:
This is *great* context - as is the other section marked context below. This definitely helps me to understand where you're at and what you want to achieve, and I think it'll be a good reminder for you as progress.